src/d3a/d3a_core/sim_results/endpoint_buffer.py

```diff
@@ -66,6 +70,23 @@ def __init__(self, job_id, initial_params, area, should_export_plots):
                 ConstSettings.GeneralSettings.EXPORT_ENERGY_TRADE_PROFILE_HR:
             self.area_market_stocks_stats = OfferBidTradeGraphStats()
 
+    @staticmethod
+    def _structure_results_from_area_object(target_area):
+        area_dict = dict()
+        area_dict['name'] = target_area.name
+        area_dict['uuid'] = target_area.uuid
+        area_dict['type'] = str(target_area.strategy.__class__.__name__)
```

Comment:
```suggestion
        area_dict['type'] = str(target_area.strategy.__class__.__name__) if target_area.strategy is not None else "Area"
```
Maybe also save the "Area" string as a global constant, so that it can be reused from other places.